events/wc_offmap_events.txt

```diff
@@ -786,4 +786,31 @@ narrative_event = {
 	option = {
 		name = EVTOPTA_WCOME.201
 	}
+}
+character_event = {
+	id = WCOME.250
+	picture = GFX_evt_cthun
+	desc = EVTDESC_WCOME.250
+	border = GFX_event_normal_frame_religion
+	
+	trigger = {
+		NOT = { religion = old_gods_worship }
+		owns = 2956j
+		b_ahnqiraj = {
+			has_building = ca_old_god_1
+		}
+	}
+	
+	mean_time_to_happen = {
```

Comment:
Should be replaced by `weight_multiplier = {`, because `mean_time_to_happen` is CPU heavy.

Reply:
Changed event to is_triggered_only